chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -319,6 +308,11 @@
    */
   public int previewCount;
 
+  /**
+   * Current time in seconds. Adjusts animated blocks like fire.
+   */
+  private double animationTime = 0;
```

Comment:
Saving and loading this value from the scene description file should also be added